### ISSUE TITLE
結果

### ISSUE THREAD

**Issue by lo-uis** [lo-uis/Team_ChoLab#3]
feature:0から
classfier:全部
training loss: 0.0045, training acc 0.9020
validation loss: 0.0047, validation acc0.8894
